src/utils/log.c

```diff
@@ -50,6 +51,24 @@ void log_init(void) {
     state->target_count = 0;
 }
 
+log_level log_level_text_to_enum(const char *level, log_level default_value) {
+    for(int i = 0; i < LOG_LEVELS; i++) {
+        if(strncmp(level_names[i], level, strlen(level_names[i])) == 0) {
```

Comment:
I guess this would let for example DEBUG12345 be DEBUG too. since comparing only up to DEBUG. Perhaps not a major problem.

Reply:
Idea was to strip out any accidental whitespace garbage